app/views/common/_scap_feature_error.erb

```diff
@@ -0,0 +1,20 @@
+<div id="host-conflicts-modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title"><%= _('OpenScap Proxy is not having Openscap feature enabled !') %></h4>
```

Comment:
Not a native speaker, but`The OpenSCAP proxy doesn't have the OpenSCAP feature enabled` sounds better to me. Note you should also properly capitalize it.